snap-build/snap.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+#
+# Copyright (c) 2018 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+# Setup the environment and build the snap image.
+# This script runs in the VM.
+
+set -x
+sudo apt-get update -y
+sudo apt-get install -y snapd snapcraft golang-go gcc g++ libcap-ng-dev pkg-config make libz-dev libssl-dev openssl cpio libelf-dev libnewt-dev \
+libiberty-dev libdw-dev libpci-dev libpixman-1-dev libglib2.0-dev librbd-dev libcap-dev libattr1-dev autoconf flex bison python docker.io
```

Comment:
Will `docker.io` clash with `docker-ce` which gets installed when you install a Kata system?

Reply:
yep, but I'm not installing Kata, I'm building Kata, and I use docker.io for osbuilder